helm/azure-admission-controller/values.yaml

```diff
@@ -1,7 +1,25 @@
+project:
+  branch: "[[ .Branch ]]"
+  commit: "[[ .SHA ]]"
+
 image:
   name: "giantswarm/azure-admission-controller"
   tag: "[[ .Version ]]"
 
-project:
-  branch: "[[ .Branch ]]"
-  commit: "[[ .SHA ]]"
+azure:
+  location: ""
+
+azureOperatorSecret:
+  service:
+    azure:
+      clientid: ""
+      clientsecret: ""
+      sptenantid: ""
+      subscriptionid: ""
+      tenantid: ""
+
+cluster:
+  baseDomain: ""
```

Comment:
Minor nitpick: Is there something behind style difference here: `clientsecret` vs. `baseDomain` - why not `clientSecret` or `basedomain` ?

Reply:
Not really. The reason is that I was using camelCase but I also copied the `clientid`, `clientsecret`, etc from `azure-operator`, and they are defined without camelCase over there. I'll change them to `clientID`, `clientSecret`, etc.